app/post-ad/page.tsx

```
@@ -313,24 +254,31 @@ export default function PostAdPage() {
-      const finalLocation = Array.isArray(formData.location) && formData.location.length === 2
-        ? `Lat: ${formData.location[0]}, Lng: ${formData.location[1]}`
-        : formData.location;
+      const finalLocation =
+        Array.isArray(formData.location) && formData.location.length === 2
+          ? `Lat: ${formData.location[0]}, Lng: ${formData.location[1]}`
+          : formData.location;
 
       const listingData = {
         title: formData.title,
         description: formData.description,
         price: parseFloat(formData.price) || null,
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)